fletcher/string_array.py

```diff
@@ -43,6 +106,24 @@ def __init__(self, array):
             )
 
 
+class Date64Array(FletcherArrayBase):
+    dtype = FletcherDtype(pa.date64())
+
+    def __init__(self, array):
```

Comment:
This could also be pushed to the base class when using #40

Reply:
Will do now